Makefile

```diff
@@ -34,7 +34,57 @@ endif
 GO=go
 GOFLAGS=""
 
+# requires:
+#	AWS_SECRET		- Secret from AWS
+#	AWS_ACCESS		- Access key from AWS
+#	S3_REGION		- Region of the bucket used to store the model
+#	S3_ENDPOINT		- Endpint of the bucket
+#	REGISTRY_USERNAME	- quay.io username
+#	REGISTRY_PASSWORD	- quay.io password
+#	REGISTRY_CREDENTIALS	- quay.io robot secret file
+go-test-setup:
+ifndef AWS_SECRET
+	$(error AWS_SECRET is undefined)
+endif
+ifndef AWS_ACCESS
+	$(error AWS_ACCESS is undefined)
+endif
```

Comment:
Agreed. Those have always been the standard names we have used across ODH workshops and tutorials since it was o[riginal ly support by the AWS SDK](https://docs.aws.amazon.com/sdkref/latest/guide/settings-reference.html#EVarSettings) so all S3 compatible APIs and will provide some level of support for those names